referral_module/api/serializers.py

```diff
@@ -0,0 +1,85 @@
+from rest_framework import serializers
+from referral_module import models
+from django.contrib.auth.models import User
+
+
+class UserSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = User
+        fields = ('id', 'username', )
+
+
+class UserReferrerSerializer(serializers.ModelSerializer):
+    class Meta:
```

Comment:
It is important to have consistent code style, e.g. having a blank line before `Meta` class definition.